wnfs/src/private/directory.rs

```diff
@@ -1150,9 +1150,18 @@ impl PrivateDirectory {
             .header
             .get_private_ref()
             .map_err(SerError::custom)?
-            .ratchet_key;
+            .revision_key;
+
+        let mut entries = BTreeMap::new();
 
-        (PrivateDirectorySerde {
+        for (name, private_ref) in self.entries.iter() {
+            let private_ref_serde = private_ref
```

Comment:
super minor: for consistency:

```suggestion
let private_ref_serializable = private_ref
```